### PR TITLE
[runtime] refine runtime style

### PR DESCRIPTION
**Pull request by robin1001**
Change-Id: Id2f9c014b9683a85ef3c401f81cf5c1135f25f44

### CODE REVIEW

File: runtime/server/x86/bin/decoder_main.cc [hunk 1]
```diff
@@ -1,17 +1,16 @@
 // Copyright 2020 Mobvoi Inc. All Rights Reserved.
 // Author: binbinzhang@mobvoi.com (Binbin Zhang)
 
-#include <glog/logging.h>
-#include <gflags/gflags.h>
-
-#include <torch/torch.h>
-#include <torch/script.h>
+#include "glog/logging.h"
```
Comment (csukuangfj) [inline runtime/server/x86/bin/decoder_main.cc:4]: Please sort them by tools, not.by.hands.

File: runtime/server/x86/bin/BUILD [hunk 2]
```diff
@@ -1,14 +1,13 @@
...
 cc_binary(
     name = "decoder_main",
     srcs = ["decoder_main.cc"],
     deps = [
         "@com_github_google_glog//:glog",
         "@com_github_gflags_gflags//:gflags",
         "@libtorch//:torch",
-        "//frontend:frontend",
         "//decoder:decoder",
+        "//frontend:frontend",
```
Comment (csukuangfj) [inline runtime/server/x86/bin/BUILD:10]: Please use tools for auto-formatting.

File: runtime/server/x86/bin/decoder_main.cc (hunk 1 above)
Comment (robin1001) [reply runtime/server/x86/bin/decoder_main.cc]: okay, fixed by clang-format

File: runtime/server/x86/bin/BUILD (hunk 2 above)
Comment (robin1001) [reply runtime/server/x86/bin/BUILD:10]: I didn't make the tool work, maybe refine it in the future

Comment (robin1001) [reply runtime/server/x86/bin/BUILD]: I read about this https://docs.bazel.build/versions/master/skylark/build-style.html, and refine it by hand.

**Comment by csukuangfj** [comment]
+1

**Review by csukuangfj — APPROVED** [review]

File: runtime/server/x86/frontend/BUILD [hunk 3]
```diff
@@ -19,8 +19,8 @@ cc_library(
     srcs = ['feature_pipeline.cc'],
     hdrs = ['fbank.h', 'feature_pipeline.h'],
```
Comment (csukuangfj) [inline runtime/server/x86/frontend/BUILD:20]: Please refer to what `machine_learning/one` and `speech/hotword` are doing on gerrit.

For consistency, a list containing more than one entries ends with a comma, like what you are doing for `deps`.

---

From https://docs.bazel.build/versions/master/skylark/build-style.html

> We let the **machine** take care of most formatting issues.

[`Buildifier`](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md), like `clang-format`, 
is just a command-line tool. Its README.md has detailed instructions for how to use it.

Don't know what blocks you.

File: runtime/server/x86/decoder/symbol_table.h [hunk 4]
```diff
@@ -33,16 +33,13 @@ class SymbolTable {
 
   ~SymbolTable() {}
 
-  std::string Find(int id) const {
-    CHECK(symbol_tabel_.find(id) != symbol_tabel_.end());
-    return symbol_tabel_[id];
-  }
+  std::string Find(int id) const { return symbol_tabel_.at(id); }
```
Comment (csukuangfj) [inline runtime/server/x86/decoder/symbol_table.h:36]: I would recommend returning `const std::string&`.

File: runtime/server/x86/frontend/BUILD (hunk 3 above)
Comment (robin1001) [reply runtime/server/x86/frontend/BUILD:20]: Yes, I just tried once following the readme and it didn't work.

Comment (robin1001) [reply runtime/server/x86/frontend/BUILD]: now fix it by hand

File: runtime/server/x86/decoder/symbol_table.h (hunk 4 above)
Comment (robin1001) [reply runtime/server/x86/decoder/symbol_table.h]: fixed

**Review by csukuangfj — CHANGES_REQUESTED** [review]
+3

**Review by csukuangfj — APPROVED** [review]

**Pull request merged**